src/partition.rs

```diff
@@ -563,16 +563,9 @@ pub fn get_intra_edges<T: Pixel>(
       needs_left = (!dc_or_cfl || x != 0)
         && !(mode == PredictionMode::D45_PRED
           || mode == PredictionMode::D67_PRED);
-      needs_topleft = mode == PredictionMode::V_PRED
-        || mode == PredictionMode::H_PRED
-        || mode == PredictionMode::D45_PRED
-        || mode == PredictionMode::D135_PRED
-        || mode == PredictionMode::D113_PRED
-        || mode == PredictionMode::D157_PRED
-        || mode == PredictionMode::D203_PRED
-        || mode == PredictionMode::D67_PRED
-        || mode == PredictionMode::PAETH_PRED;
-      needs_top = !dc_or_cfl || y != 0;
+      needs_topleft =
+        mode.is_directional() || mode == PredictionMode::PAETH_PRED;
+      needs_top = (!dc_or_cfl || y != 0) && mode != PredictionMode::D203_PRED;
```

Comment:
> && mode != PredictionMode::D203_PRED
Why is this added?

Reply:
This angle mode is not required for "needs_top" and this does not affect to the output.

Reply:
Can you specify where this is mentioned, e.g. spec section, page, or reference to libaom?

Reply:
"AboveRow" is not required "if pAngle is greater than 180" as
in, 7.11.2.4.Directional intra prediction process (9.)
[AV1 spec](https://aomediacodec.github.io/av1-spec/av1-spec.pdf)